src/components/Image/Image.jsx

```diff
@@ -46,10 +50,13 @@ function Image ( {
 	srcSet,
 	title,
 } ) {
+	const classes = `${styles.image} ${className}`;
```

Comment:
cleaner markup. or use cx

```suggestion
	const classes = className ? `${styles.image} ${className}` : styles.image;
```